docs/rafter/11-02-asyncapi-service.md

```diff
@@ -0,0 +1,31 @@
+---
+title: AsyncAPI Service
+type: Metrics
+---
+
+This table shows the AsyncAPI Service custom metrics, their types, and descriptions.
+
+| Name | Type | Description |
+|------|-------------|------|
+| `rafter_services_http_request_and_mutation_duration_seconds` | histogram | Specifies a number of assets that the service received for processing and mutated within a given time series. |
```

Comment:
Should it be `a` number or `the` number in this case and in the following rows? I think it should be `the number` as we point to the specific number of assets. 🤔